### ISSUE TITLE
N2 ignition point should be 20%

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/f16#122]
https://github.com/JSBSim-Team/jsbsim/issues/233

prevents this from being fixed

**Comment by NikolaiVChr** [comment]
Changing this issue, to be about RPMs in general

**Comment by NikolaiVChr** [comment]
From mark on f-16.net on max N2:
F-110-GE100 Max RPM 106%
F-100-PW220 Max RPM 96 %
F-100-PW200 Max RPM 96%
F-110-GE129 Max RPM 108%
F-100-PW229 Max RPM 97%

and on rpms:

F-110-GE100 100% on the gauge is equiv to 14,460 RPM
F-100-PW220 100% on the gauge is equiv to 14,000 RPM

**Comment by NikolaiVChr** [comment]
for F100-PW-229:

100% N1 is 10,000 and 100% N2 = 14,000
MAXIMUM is 11,000 for N1 and 13,800 for N2.

**Comment by NikolaiVChr** [comment]
Timi found this doc:
https://www.nasa.gov/centers/dryden/pdf/88434main_H-2122.pdf

The paper test the F110-GE-129:

Idle power is at 20° power lever angle (PLA) setting, and intermediate (maximum nonafterburning) power is at 85° PLA. [What is max PLA????]

In that paper the rpm reached:
N1  8520
N2  15094

**Comment by NikolaiVChr** [comment]
Furthermore the Tachometer is different on GE engines.
GE goes to 110%

**Comment by NikolaiVChr** [comment]
PLA for F100:
0 degrees - cutoff
16 degrees - Idle
86 degrees - Intermediate or Military
91 degrees - Min AB
130 degrees - Max AB

**Comment by NikolaiVChr** [comment]
> the engine schedule is intended for a relatively linear relationship between PLA and thrust at any particular flight condition

Interesting, will have to make that..

**Comment by NikolaiVChr** [comment]
06760e2d526330d0f2f85b971c8514a3285ec247 fixes linear relationship and indent position for F100 throttle.

**Comment by NikolaiVChr** [comment]
Timi: According to this article the later blocks can go from idle to afterburner in two seconds:
https://www.codeonemagazine.com/article.html?item_id=23

**Comment by JMaverick16** [comment]
BTW I still think our spooling time from 0 to 25% is slower compared to videos of it. Thoughts?

**Comment by NikolaiVChr** [comment]
Time to consider fixing this? Users of older FG, will have to use older versions of F-16.

**Comment by JMaverick16** [comment]
Older FG as in < 2020.2?

**Comment by NikolaiVChr** [comment]
No, as in less than 2020.1

**Comment by JMaverick16** [comment]
_J Mav approves_

**Comment by ph-jake** [comment]
> BTW I still think our spooling time from 0 to 25% is slower compared to videos of it. Thoughts?

Looking at the "F-16 Engine run, Iraq" video, time from Start 2 to JFS Run light is about 8 seconds, then another 20 seconds for the RPMs to rise to 20%. Our simulation takes 30 seconds to JFS run light, and about 10 seconds for RPMs to 20%.

**Comment by NikolaiVChr** [comment]
Yes, they probably improved the JFS since the one I modeled, I think that was a rather old model.
Also I do believe it spools up faster when doing START 2 than START 1, but I seem to remember I could not find info how how much faster.

As for getting to 20% that might vary alot on the engine installed. But we can change that rate, but wont be supported for 2018.3, so I think this should wait till 2020.3 which should be in a couple of days.

**Comment by NikolaiVChr** [comment]
N1/N2 ignition is now 5%/20% for 2020.3
Which means ignition is now wrongly 20%/25% for 2019.1

**Comment by NikolaiVChr** [comment]
In the same video from Iraq (block 30), I measured this:
`<n2startrate>     1.6 </n2startrate>`
Thats the spooling speed from 20 to idle.
I haven't put in that number though, as it should match JFS and 0 to 20 also.
Maybe we should have a JFS spool time for each block.

**Comment by JMaverick16** [comment]
Wouldn't be a bad idea. I support it

**Comment by ph-jake** [comment]
> In the same video from Iraq (block 30), I measured this:
> `<n2startrate> 1.6 </n2startrate>`
> Thats the spooling speed from 20 to idle.
> I haven't put in that number though, as it should match JFS and 0 to 20 also.

I don't get it. When the JFS is switched on (START 2 or START 1) the bottle(s) blow and get the JFS up to run speed ASAP. Timing the video shows from switch on to run light < 10 seconds, using START 2 (both bottles). Only then the engine starter is engaged, using the JFS energy to spin the core up to 20% for ignition.
What does the n2startrate have to do with the the time the JFS spools up?

**Comment by NikolaiVChr** [comment]
Nobody said they are related.
But this issue is about RPM in general, both JFS and engine.
Please reread my post, I make no claim to talk about JFS in regards to that value.

**Comment by NikolaiVChr** [comment]
..and when I say 'match' I mean not they should be same.

**Comment by NikolaiVChr** [comment]
Let me detail what I meant by 'match':

If we just fix engine spool-up from 0 to 20 and from 20 to idle, without changing JSF also, starting total is going to take alot longer than now and that it does in real life.

So need more info on different JFS spoolup times than just 8s with START 2 from block 30.
..or we need to guesstimate.

All we know now is one JFS (which blocks?) takes 30s and a block 30 (MLU?) takes 8s.

**Comment by NikolaiVChr** [comment]
Alright, I changed the JFS, it can now be configured per block.

I put in these numbers:
```
Total START 2 startup times:  (JFS + to 20 + to idle)
block YF/10/20   : 20+20+25 = 65s
block 30/40/50/60:  8+20+25 = 53s

Total START 1 startup times:
block YF/10/20   : 30+20+25 = 75s
block 30/40/50/60: 12+20+25 = 57s
```

I had to increase autostart watchdog timers to be sure it could handle block 10 (START 1)

**Comment by NikolaiVChr** [comment]
Some engines seems to have 65% idle. I am guessing its the GE engines.
